device/celestica/x86_64-cel_e1031-r0/sonic_platform/common.py

```diff
@@ -195,7 +185,7 @@ def write_txt_file(self, file_path, value):
         return True
 
     def is_host(self):
-        return os.system(self.HOST_CHK_CMD) == 0
+        return subprocess.call(self.HOST_CHK_CMD) == 0

```

Comment:
If use subprocess.call, need to suppress stdout and stderr and handle docker command not found exception in docker space.
The following example codes work in both host and docker mode.
``` python3
def is_host():
    try:
        subprocess.call(HOST_CHK_CMD, stdout=subprocess.DEVNULL,
            stderr=subprocess.DEVNULL)
    except FileNotFoundError:
        return False
    return True
```

Reply:
I think os.system is equivalent with subprocess.call without specifying stdour and stderr to DEVNULL. Ref: https://docs.python.org/3/library/subprocess.html#replacing-os-system

Reply:
I got what you meant, I will change per your suggestion

Reply:
> I think os.system is equivalent with subprocess.call without specifying stdout and stderr to DEVNULL. Ref: https://docs.python.org/3/library/subprocess.html#replacing-os-system

Yes, no essential differences between the two functions, but HOST_CHK_CMD had been changed from "docker > /dev/null 2>&1" to ["docker"], so there is no way to suppress the 'docker' command output any more in host mode. And handling the FileNotFoundError exception is also necessary for docker mode.

Reply:
Fixed. Could you review and approve? Thank you!